environment.yml

```diff
@@ -0,0 +1,169 @@
+name: null
+channels:
+  - openeye
+  - file:///apps/rocs/restricted/pRED/testing/cascadelake/software/csd-python-api/3.0.9
```

Comment:
Is this folder location going to exist on everyone's computer if they clone the repo and follow the setup instructions?